sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/TestPojo.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * POJO Class for usage in tests
+ */
+public class TestPojo {
```

Comment:
Suggestion: There are some test pojos like Person, City, Pet etc in `com.azure.cosmos.rx.pojos`. See if you can reuse them or if you want to move this class to comsos.pojos package.

Reply:
Name is not generic, wont qualify in other test